.github/workflows/tsdoc.yml

```
@@ -0,0 +1,107 @@
+            <meta http-equiv="refresh" content="0; url=./${{ github.event.release.tag_name }}/">
+            <title>Redirecting to ${{ github.event.release.tag_name }}</title>
+          </head>
+          <body>
+            <p>Redirecting to <a href="./${{ github.event.release.tag_name }}/">${{ github.event.release.tag_name }}</a>...</p>
+          </body>
+          </html>
+          EOF
+          fi
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
```

actionlint (1.7.7)

the runner of "actions/configure-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)